modules/hcp-migrate-aws-single-to-multiarch.adoc

```diff
@@ -0,0 +1,168 @@
+= Transitioning a hosted cluster from single-architecture to multi-architecture
+
+You can transition your single-architecture 64-bit AMD hosted cluster to a multi-architecture hosted cluster on {aws-first} to reduce the cost of running your workloads. For example, you run your existing workloads on 64-bit AMD while transitioning it to 64-bit ARM. You can manage your workloads from a central Kubernetes cluster.
+
+A single-architecture hosted cluster can manage node pools of only one particular CPU architecture. However, a multi-architecture hosted cluster can manage node pools with different CPU architectures. On {aws-short}, a multi-architecture hosted cluster can manage both 64-bit AMD and 64-bit ARM node pools.
+
+.Prerequisites
+
+Before transitioning a hosted cluster from single-architecture to multi-architecture, review the following requirements:
+
+* You have installed an {product-title} management cluster for {aws-short} on {rh-rhacm-first} with the {mce}.
+* You have an existing single-architecture hosted cluster that uses 64-bit AMD variant of the {product-title} release payload.
+* An existing node pool that uses the same 64-bit AMD variant of the {product-title} release payload and is managed by the existing hosted cluster.
+* Ensure that you installed the following command-line tools:
+
+** `oc`
+** `kubectl`
+** `hcp`
+** `skopeo`
+
+.Procedure
+
+. Review the existing {product-title} release image of the single-architecture hosted cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get hostedcluster/<hosted_cluster_name> -o jsonpath='{.spec.release.image}'
+----
++
+.Example output
+[source,terminal]
+----
+quay.io/openshift-release-dev/ocp-release:4.16.5-x86_64
+----
+
+. In your {product-title} release image, if you use the digest instead of the tag, find the multi-architecture tag version of your release image:
+
+.. Get the {product-title} version and save it in the `OCP_VERSION` variable by running the following command:
++
+[source,terminal]
+----
+OCP_VERSION=$(oc image info quay.io/openshift-release-dev/ocp-release@sha256:ac78ebf77f95ab8ff52847ecd22592b545415e1ff6c7ff7f66bf81f158ae4f5e -o jsonpath='{.config.config.Labels["io.openshift.release"]}')
+----
+
+.. Get the multi-architecture tag version of your release image and save it in the `MULTI_ARCH_TAG` variable by running the following command:
++
+[source,terminal]
+----
+MULTI_ARCH_TAG=$(skopeo inspect docker://quay.io/openshift-release-dev/ocp-release@sha256:ac78ebf77f95ab8ff52847ecd22592b545415e1ff6c7ff7f66bf81f158ae4f5e | jq -r '.RepoTags' | sed 's/"//g' | sed 's/,//g' | grep -w "$OCP_VERSION-multi$" | xargs)
+----
+
+.. Save the multi-architecture release image name in the `IMAGE` variable by running the following command:
++
+[source,terminal]
+----
+IMAGE=quay.io/openshift-release-dev/ocp-release:$MULTI_ARCH_TAG
+----
+
+.. Get the list of the multi-architecture image digests, by running the following command:
```

Comment:
```suggestion
.. To see the list of the multi-architecture image digests, by running the following command:
```

Just suggesting to change `Get` to `see` since you don't really need this step for the migration.